assets/js/base/components/checkbox-control/style.scss

```diff
@@ -7,37 +7,45 @@
 
 	.wc-block-components-checkbox__input[type="checkbox"] {
 		appearance: none;
-		border: 1px solid currentColor;
-		height: 1em;
+		border: 2px solid currentColor;
+		border-radius: 2px;
+		height: rem(18px);
+		width: rem(18px);
 		margin: 0;
-		min-height: 16px;
-		min-width: 16px;
+		min-height: 18px;
+		min-width: 18px;
 		overflow: hidden;
 		position: static;
 		vertical-align: middle;
-		width: 1em;
+		background-color: #fff;
 
 		&:checked {
 			background: currentColor;
 			border-color: currentColor;
+		}
 
-			&::before {
-				color: #fff;
-				content: "\2713";
-				display: block;
-				height: calc(1rem - 2px);
-				min-height: 14px;
-				min-width: 14px;
-				margin: 0;
-				line-height: 100%;
-				text-align: center;
-				width: calc(1rem - 2px);
-			}
+		&:not(:checked) + .wc-block-components-checkbox__mark {
+			display: none;
 		}
 	}
 
-	.wc-block-components-checkbox__input[type="checkbox"] + .wc-block-components-checkbox__label {
+	.wc-block-components-checkbox__mark {
+		fill: #fff;
+		position: absolute;
+		left: rem(1px);
+		top: rem(5px);
+		width: 16px;
+		height: 16px;
```

Comment:
It looks like this wasn't solved. `width` and `height` should be `em()`. Otherwise this is how it looks like if I set my browser default font size to 32px:
![imatge](https://user-images.githubusercontent.com/3616980/89875869-1624c880-dbbe-11ea-990a-4d6f5602fe04.png)